data-prepper-expression/src/main/java/org/opensearch/dataprepper/expression/CidrExpressionFunction.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.expression;
+
+import inet.ipaddr.IPAddress;
+import inet.ipaddr.IPAddressString;
+import org.opensearch.dataprepper.model.event.Event;
+
+import javax.inject.Named;
+import java.util.List;
+import java.util.Objects;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+@Named
+public class CidrExpressionFunction implements ExpressionFunction {
+
+    @Override
+    public String getFunctionName() {
+        return "isIPinCIDR";
+    }
+
+    @Override
+    public Object evaluate(final List<Object> args, Event event, Function<Object, Object> convertLiteralType) {
+        if (args.size() <= 1) {
+            throw new IllegalArgumentException("isIpInCidr() takes at least two arguments");
```

Comment:
This name is inconsistent with the `getFunctionName()`. I recommend making this a constant to use throughout.